fail2ban/server/utils.py

```diff
@@ -50,34 +50,41 @@ class Utils():
 	DEFAULT_SLEEP_INTERVAL = 0.01
 
 
-	class Cache(dict):
+	class Cache(object):
```

Comment:
Ah now see your point in commit message...
Well, I think that inheritance to dict in python is "safe" enough (other slots are in __dict__), but OK (because really not whole interface overwritten).